### PR TITLE
Checked whether balanced Pods were created.

### LINKED ISSUE #47382: Test Failure: Pods created by ReplicationController should spread to different node
[jenkins](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-cvm-1-6-gci-1-7-upgrade-cluster-new/6#k8sio-schedulerpriorities-serial-pods-created-by-replicationcontroller-should-spread-to-different-node)

[test-grid](https://k8s-testgrid.appspot.com/1.6-1.7-upgrade#gke-cvm-gci-upgrade-cluster-new&include-filter-by-regex=%5EOverall%24%7C%5C%5Bk8s%5C.io%5C%5D%5C%20SchedulerPriorities%5C%20%5C%5BSerial%5C%5D%5C%20Pods%5C%20created%5C%20by%5C%20ReplicationController%5C%20should%5C%20spread%5C%20to%5C%20different%5C%20node)

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduling/priorities.go:154
Jun 12 20:54:43.472: Pods are not spread to each node.
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduling/priorities.go:152
```

Chronically failing test.  Perhaps covers deprecated behavior?

/sig scalability 
/assign @cheftako

### PR DESCRIPTION
**Pull request by k82cn**
**What this PR does / why we need it**:
In #47382, it seems balanced Pods were not created; this PR will check whether balanced Pods were created, and show logs.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: partially fixes #47382 

**Release note**:

```release-note-none
```

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@k82cn Thanks
/lgtm

File: test/e2e/scheduling/priorities.go [hunk 1]
```diff
@@ -163,7 +165,9 @@ var _ = framework.KubeDescribe("SchedulerPriorities [Serial]", func() {
 		defer framework.RemoveLabelOffNode(cs, nodeName, k)
 
 		// make the nodes have balanced cpu,mem usage ratio
-		createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.6)
+		err := createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.6)
+		framework.ExpectNoError(err)
+
```
Comment (gyliu513) [inline test/e2e/scheduling/priorities.go]: I found that most code do not include a blank line for such case, ditto for others

**Comment by timothysc** [comment]
please address the minor formatting comment then lgtm, pretty straight forwards here.

**Comment by k82cn** [comment]
@timothysc , do you think we should merge it for 1.7? so we can get more logs for flaky test.

**Comment by timothysc** [comment]
At this point, no.

**Comment by k82cn** [comment]
sure, np :).

**Comment by k82cn** [comment]
ping for /lgtm .

**Comment by k82cn** [comment]
rebased, @timothysc , would you help to add lgtm again :).

**Pull request merged**